tests/spec_decode/e2e/test_mlp_correctness.py

```diff
@@ -34,7 +34,7 @@
 MAX_SPEC_TOKENS = 5
 
 # precision
-PRECISION = "float16"
+PRECISION = "float32"
```

Comment:
@njhill @cadedaniel 

Would you be okay with this change?

Reply:
Welp, now we are OOM in the CI

Reply:
Will dive deeper

Reply:
Nick is making another model FYI

Reply:
thanks @njhill !